controller/src/test/java/org/jboss/as/controller/persistence/PersistanceResourceTestCase.java

```diff
@@ -370,6 +371,15 @@ public void testRelativePathFromOutsideConfigDirectory() throws Exception {
         }
     }
 
+    @Test
+    public void testSymbolicLinkForConfigurationFiles() throws Exception {
+        Path file = Paths.get(createFile(externalDir, "standard.xml", "test").toString());
+        Path symbolicLink = Paths.get(standardDir + "standard.xml");
```

Comment:
The `standardDir` is a file and it is not trailed with a slash, so the `symbolicLink` is not being created correctly.

You should use something like `Path symbolicLink = standardDir.toPath().resolve("standard.xml");`

Another issue with the test is you are never deleting the `standard.xml` file located by default on the `standardDir`. You need to replace this file with the symbolic link